titiler/xarray/factory.py

```diff
@@ -18,6 +20,78 @@
 from titiler.core.resources.responses import JSONResponse
 
 
+def xarray_open_dataset(
```

Comment:
move them outside the factory for now, but we can add them back later as dependencies

Reply:
sorry - do you mean add these functions outside of the factory, to another module, and then import them here?

Reply:
for now we can keep them in `factory.py` but there is no real need to have them in the `XarrayTilerFactory`